x-pack/plugins/security_solution/public/common/components/top_n/index.test.tsx

```diff
@@ -251,8 +251,8 @@ describe('StatefulTopN', () => {
     test('it has a combinedQueries value from Redux state composed of the timeline [data providers + kql + filter-bar-filters] when rendering in a timeline context', () => {
       const props = wrapper.find('[data-test-subj="top-n"]').first().props() as Props;
 
-      expect(props.combinedQueries).toEqual(
-        '{"bool":{"must":[],"filter":[{"bool":{"filter":[{"bool":{"should":[{"match_phrase":{"network.transport":"tcp"}}],"minimum_should_match":1}},{"bool":{"should":[{"exists":{"field":"host.name"}}],"minimum_should_match":1}}]}},{"match_phrase":{"source.port":{"query":"30045"}}}],"should":[],"must_not":[]}}'
+      expect(props.combinedQueries).toMatchInlineSnapshot(
```

Comment:
nit: I think this would be better if it was something which matched against an object rather than against a string.

Something like: `expect(JSON.parse(props.combinedQueries)).toMatchInlineSnaphshot(...)` if that works out. Otherwise this looks very hard to read and when people run the update jest command they're not going to look to see if they are fixing anything or not. But rather just mindlessly run it and do a commit and then the reviewer will have a hard time to read it since the string isn't being indented.